### PR TITLE
Capstone project first module

### PR DESCRIPTION
**Pull request by IngDavidTM**
### Biotechnology Congress

This is the Capstone project. It has two pages for the mobile version and a mobile menu created using JavaScript. Also, this project has a desktop version to include more users. Finally, the Featured Speakers section was designed using JavaScript.

It uses:

- `linters`
- `CSS`
- `JavaScript`
- `HTML`

Video: [Page description by David Tamayo](https://www.loom.com/share/c988a08fa2d2490387077c10e816a6a3)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Science
+- Nature
+- Cell Press
+- JABB
```
Comment (Bondok6) [inline README.md:59]: - [ ] Your README is well documented 💯 but according to the requirements, I would suggest giving appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) :wink: :pushpin:

File: about.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    }
+  }
+  openWrap.addEventListener('click', wrapFunction);
+  closeWrap.addEventListener('click', wrapFunction);
+  const imgHomeSetup = document.querySelectorAll('.img-home-setup');
+  function homeFunction() {
+    window.location.href = 'index.html';
+  }
+  imgHomeSetup.forEach((img) => { img.addEventListener('click', homeFunction); });
+});
```
Comment (Bondok6) [inline about.js]: - [ ] I noticed that you have exactly the same code in `index.js` file, So I suggest deleting this file and you can use `  <script src="index.js"></script>` in your `about.html` page instead 💡

> To follow [JavaScript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) and to Keep your [code clean](https://devinduct.com/blogpost/22/javascript-clean-code-best-practices) 💯

File: style.css [hunk 3]
```diff
@@ -0,0 +1,676 @@
...
+
+.main-program h4 {
+  color: #ec5242;
+  font-size: 16px;
+  min-width: 72px;
+}
+
+.main-program p {
+  font-size: 12px;
+}
```
Comment (Bondok6) [inline style.css]: - [ ] Your design is awesome 💯 However. I suggest using `text-align: center` here to center the paragraph to follow the original design layout 👌

> According to the requirements, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), Layout: composition and space between elements.

| Your Design  | Original Design  |   
| --- | --- |
|   |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @IngDavidTM 🙋‍♂️, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!



## To Highlight 
- Great job using semantic HTML5 tags ✅
- No linter errors ✅
- Your repo is professional ✅
- Your project is professional ✅
- Great job implementing Additional requirements (More button) ✅
- Good video presentation ✅

## Suggested changes

- [x] _Check the comments under the review._ ⬇

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            <article>
+                <img src="images/lecture5.png" alt="">
+                <h4>BC Ignite</h4>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio maiores</p>
+            </article>
+        </div>
+        <button>Join The web Development Course</button>
+    </section>
+    <section class="partner" id="partner">
+        <h2>Partner</h2>
```
Comment (Kamba56) [inline index.html]: - [Optional] Please use an indentation of two spaces for readability in all your documents. This is best practice and also a convention. Check [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more details, please.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+                <div>
+                    <a href="about.html">About</a>
+                    <a href="">Program</a>
+                    <a href="">Join</a>
+                    <a href="">Sponsor</a>
+                    <a href="">News</a>
+                    <a href="">BC Campaign</a>
+                </div>
+            </div>
+        </nav>
```
Comment (Kamba56) [inline index.html]: - [Optional] Please consider wrapping this part of your document in the `<header>` element. This is best for HTML semantics.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+        <div class="presentation-container">
+            <header>
+                <h3>"Hello! Science world"</h3>
+                <h1>WORLD BIOTECH CONGRESS 2022</h1>
+            </header>
```
Comment (Kamba56) [inline index.html]: - [Optional] consider using a `<div>` to wrap this since this is not the header of your document.😁

**Review by Kamba56 — APPROVED** [review]
Hi @IngDavidTM, 

You have done a great job. Your project is wonderful, keep up the good work.
In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
IThinkYouShouldBeReallyProudOfYourselfLeonaLewisGIF
FriendsJoeyGIF


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**